README.md

```diff
@@ -96,146 +44,46 @@ After you're finished please remove all the comments and instructions!
   </ul>
 </details>
 
-<!-- Features -->
-
 ### Key Features <a name="key-features"></a>
 
-> Describe between 1-3 key features of the application.
-
-- **[key_feature_1]**
-- **[key_feature_2]**
-- **[key_feature_3]**
+- **[Relational data base]**
+- **[Queries]**
 
 <p align="right">(<a href="#readme-top">back to top</a>)</p>
 
-<!-- LIVE DEMO -->
-
-## 🚀 Live Demo <a name="live-demo"></a>
-
-> Add a link to your deployed project.
+## 🚀 Live Demo <a name="N/A"></a>
 
-- [Live Demo Link](<replace-with-your-deployment-URL>)
+- [Not Available](<N/A>)
```

Comment:
- [ ] Kindly fix this broken link, this will make your README file to be more professional and engaging. 👍 **You can remove this header if there is no live deployment at the moment** 🙏